code/Language/Drasil/Chunk/ShortName.hs

```diff
@@ -9,6 +11,7 @@ class HasShortName  s where
                             -- A short name used for referencing within a document that can 
                             -- include symbols and whatnot if required.
                             -- Visible in the typeset documents (pdf)
+  shortNm :: Lens' s ShortName
```

Comment:
We really don't need both `shortname` and `shortNm`. Especially since `shortname` should always be equal to `view shortNm`.

Reply:
@JacquesCarette Hello Dr. Carette, yeah I realized that having both is unnecessary but creating an instance of `HasShortName` is much easier with a Lens.
In that case, should I remove `shortname` or would you prefer to keep the getter?

Reply:
I would then make 'shortname' be that Lens.

Reply:
Since I'll be removing the getter function and only have a Lens, should I go through in a separate branch first and add a `Label` to all the Chunk types?

Reply:
You should not be adding a `Label` to *all* the chunk types.  Only the ones that need it.  Certainly all the ones that currently have a short name do. But that's probably it.

Having said that, yes, it is probably best to first switch from explicit short name to using label. And then only labels should have short names.